index.js

```diff
@@ -289,8 +290,8 @@ function choosePluralForm(text, locale, count) {
 // interpolation placeholder.
 var dollarRegex = /\$/g;
 var dollarBillsYall = '$$';
-var tokenRegex = /%\{(.*?)\}/g;
-function interpolate(phrase, options) {
+function interpolate(phrase, options, tokenPattern) {
+  var tokenRegex = new RegExp(tokenPattern, 'g');
```

Comment:
Hm, sure! Perhaps just `prefix` and `suffix` for now, though. However, we'd still have to escape any special regex characters to avoid accidental crashes.